src/bindings/js/node/tests/unit/model.test.js

```diff
@@ -221,7 +225,82 @@ describe('ov.Model tests', () => {
       );
     });
 
-    // TODO: Test reshape of model with multiple inputs
+    it('test reshape with ports', () => {
+      const model = core.readModelSync(getAddModel());
+      const newShape = new ov.PartialShape('1, 4');
+      for (const modelInput of model.inputs) {
+        assert.ok(modelInput instanceof ov.Output);
+        model.reshape(new Map([
+          [modelInput, newShape],
+        ]));
+        assert.deepStrictEqual(
+          modelInput.getPartialShape().toString(), newShape.toString());
+      }
```

Comment:
Why do we call the model reshape from a loop? Let's create a map object and run it only one once.
I suppose this is also valid for other test cases.

Reply:
I changed tests to call `model.reshape` once per test. I believe checking inputs in a loop is fine ;)